### DISCUSSION TITLE
Unable to run dbeaver on windows 10 32bit

### DISCUSSION THREAD

**Discussion by giopappy** [dbeaver/dbeaver#10816] [Q&A] [answered]
Hallo, I've downloaded latest version (7.3.1) but it seems it's not compatible with my windows 10 32bit installation (it's a test virtual machine).
Checking the old versions I see the 5.3.5 is the latest which support 32bit architecture. Is this right?
There's a way to let 7.3.1 version to be executed over a 32bit so?

Thanks

**Comment by uslss** [comment, marked as answer]
We stopped delivering 32-bit packages of DBeaver CE since 6.0.1. Mostly because Eclipse platform stopped supporting 32-bit and because Java stopped this was well.
